torch/csrc/jit/passes/quantization_patterns.h

```diff
@@ -88,13 +88,29 @@ graph(%a_quant, %packed_params, %r_scale, %r_zero_point, %r_dtype, %stride, %pad
         %r_quant = quantized::conv2d_relu(%a_quant, %packed_params, %stride, %padding, %dilation, %groups, %r_scale, %r_zero_point)
         return (%r_quant) )";
 
+  std::string inplace_add_relu = R"(
```

Comment:
What about non-inplace add relu? Do you need a new pattern for that? Also, can you add a test for this pattern?

Reply:
yes, we need another pattern for non inplace add relu